hail_search/queries/base.py

```diff
@@ -273,6 +277,9 @@ def import_filtered_table(self, sample_data, intervals=None, **kwargs):
             entry_type = families_ht.family_entries.dtype.element_type
             for project_ht, num_project_families in filtered_project_hts[1:]:
                 families_ht = families_ht.join(project_ht, how='outer')
+                families_ht = families_ht.annotate_globals(
```

Comment:
Is this using the `family_guids` global that's on the families table?  I think that it's actually something I was using internally (to pass the list of loadable families between tasks) and don't think it's reliable.

Reply:
👍 I'm going to remove it from the pipeline output then.